compat/vcbuild/vcpkg_install.bat

```diff
@@ -34,6 +34,12 @@ REM ================================================================
 	@FOR /F "delims=" %%D IN ("%~dp0") DO @SET cwd=%%~fD
 	cd %cwd%
 
+	SET arch=x64-windows
+
+REM is vcpkg package installed and complied \\compat\\vcbuild\\vcpkg\\installed\\\$(VCPKGArch)\\include\\openssl\\ssl.h
```

Comment:
Please fix the indentation.

Reply:
> indentation

The current file uses single space after the REM. Was there an issue with the existing file, or is there something I can't see in the Github gui?